### PR TITLE
Added plugin template and scripted test

### PR DESCRIPTION
**Pull request by alexmihailov**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -1,4 +1,10 @@
 lazy val `sbt-maven-plugin` = (project in file(".")).enablePlugins(SbtWebBase)
+  .settings(
+    scriptedLaunchOpts := {
+      scriptedLaunchOpts.value ++ Seq("-Xmx1024M", "-Dplugin.version=" + version.value)
```
Comment (mkurz) [inline build.sbt:4]: `-Xmx1024M` seems a bit high, I think 256 should be enough. Maybe try 128 first.

**Comment by mkurz** [comment]
To run the scripted tests with GitHub actions have a look at https://github.com/playframework/playframework/blob/63c8845ca6392094f079c35652a04a12b153d779/.github/workflows/build-test.yml#L145-L178
The most relevant part is: https://github.com/playframework/playframework/blob/63c8845ca6392094f079c35652a04a12b153d779/.github/workflows/build-test.yml#L168-L176

File: build.sbt (hunk 1 above)
Comment (alexmihailov) [reply build.sbt]: Hi @mkurz ! Works successfully with 128

**Comment by alexmihailov** [comment]
> To run the scripted tests with GitHub actions have a look at https://github.com/playframework/playframework/blob/63c8845ca6392094f079c35652a04a12b153d779/.github/workflows/build-test.yml#L145-L178 The most relevant part is: https://github.com/playframework/playframework/blob/63c8845ca6392094f079c35652a04a12b153d779/.github/workflows/build-test.yml#L168-L176

Made a separate job to run scripted test

**Review by ihostage — APPROVED** [review]
LGTM.
I think this is a easiest and understandable first step.
@mkurz WDYT? Can we merge that?

**Review by mkurz — CHANGES_REQUESTED** [review]
One thing, after that merge ready :wink:

File: src/sbt-test/sbt-maven/simple/project/build.properties [hunk 2]
```diff
@@ -0,0 +1 @@
+sbt.version=1.9.0
```
[the remark this answers is not in the document]
Comment (alexmihailov) [reply src/sbt-test/sbt-maven/simple/project/build.properties]: Hi @mkurz ! I fixed this

**Pull request merged**